website/content/docs/enterprise/federation.mdx

```diff
@@ -10,9 +10,9 @@ description: >-
 # Consul Enterprise Advanced Federation
 
```

Comment:
This is kind of a pointless page, regardless of the link to the matrix. Users are going to be frustrated that there isn't information about how to use the feature. We don't even link to the applicable information in the docs (which I believe is the [Service-to-service Traffic Across Partitions](https://www.consul.io/docs/connect/gateways/mesh-gateway/service-to-service-traffic-partitions#service-to-service-traffic-across-partitions) page). My recommendation is to work the appropriate content into the core docs and link from the core docs to the matrix.

Reply:
I agree, though I think making this page more useful is out of scope for this PR.

This reminds me of how the network segments docs page [used to be](https://www.consul.io/docs/v1.9.x/enterprise/network-segments) versus [how it is now](https://www.consul.io/docs/enterprise/network-segments). I think we can make it more useful going forward (like we did for the network segments page).

I do have other concerns about this page. For example, it's ostensibly about the network areas feature, but network areas isn't the title of the page or how it's described in the feature list. This page doesn't come up at all if you search "network areas" on the site (though maybe that's actually OK for now because the current state of the page is not that helpful!)